### PR TITLE
Fix ugly exception time marker in trace UI

### PR DESCRIPTION
**Pull request by daniellok-db**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/daniellok-db/mlflow/pull/14005?quickstart=1)

#### Install mlflow from this PR

```
# Use `%sh` to run this command on Databricks
OPTIONS=$(if pip freeze | grep -q 'mlflow @ git+https://github.com/mlflow/mlflow.git'; then echo '--force-reinstall --no-deps'; fi)
pip install $OPTIONS git+https://github.com/mlflow/mlflow.git@refs/pull/14005/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 14005
```

</p>
</details>

### Related Issues/PRs

 #xxx

### What changes are proposed in this pull request?

There was an improper flag set, causing the tag background color to be the same as the text. Also this fixes the problem where there are 700+ console warnings when viewing the UI:

https://github.com/user-attachments/assets/2ff99f4b-2fe1-4d58-b035-c54765d77c2b

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests

Now:

https://github.com/user-attachments/assets/4a4be044-76c1-4adb-8b3e-90a6dfefa7e4

https://github.com/user-attachments/assets/0f9e51ec-99b4-4d70-a9aa-148181ec297a

### Does this PR require documentation update?

- [x] No. You can skip the rest of this section.
- [ ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/deployments`: MLflow Deployments client APIs, server, and third-party Deployments integrations
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

#### Should this PR be included in the next patch release?

`Yes` should be selected for bug fixes, documentation updates, and other small changes. `No` should be selected for new features and larger changes. If you're unsure about the release classification of this PR, leave this unchecked to let the maintainers decide.

<details>
<summary>What is a minor/patch release?</summary>

- Minor release: a release that increments the second part of the version number (e.g., 1.2.0 -> 1.3.0).
  Bug fixes, doc updates and new features usually go into minor releases.
- Patch release: a release that increments the third part of the version number (e.g., 1.2.0 -> 1.2.1).
  Bug fixes and doc updates usually go into patch releases.

</details>

- [x] Yes (this PR will be cherry-picked and included in the next patch release)
- [ ] No (this PR will be included in the next minor release)

### CODE REVIEW

File: mlflow/server/js/src/experiment-tracking/components/traces/TraceDataDrawer.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ export const TraceDataDrawer = ({
       return (
         <div
           css={{
-            height: '100%',
+            height: `calc(100% - ${theme.spacing.sm}px)`,
```
Comment (daniellok-db) [inline mlflow/server/js/src/experiment-tracking/components/traces/TraceDataDrawer.tsx:157]: fix a small scrolling issue:


https://github.com/user-attachments/assets/018d0f1e-f36a-41ad-bf50-e4a41ae5ec6d

**Comment by B-Step62** [comment]
@daniellok-db Would you educate me what is the proper way to test this in local? I have no idea for generated contents, so trying to validate it locally, but still sees the same color after installing the PR and start a new UI with hard refresh.

**Comment by daniellok-db** [comment]
@B-Step62 hmm that's strange, are you running the JS dev server at `localhost:3000`, or using the built UI assets and just using `mlflow server`?

**Comment by daniellok-db** [comment]
@B-Step62 if you `cd mlflow/server/js && yarn && yarn build`, and then go to `localhost:5000` instead, does it work?

**Comment by B-Step62** [comment]
@daniellok-db I'm usiing `yarn build & yarn start` -> `localhost:3000` but still seeing the old one... (also removed `build` file). Perhaps the issue is in my side.

**Comment by daniellok-db** [comment]
@B-Step62 i think it should be a caching issue but i'm not sure how to resolve it on your end—this shouldn't be an issue for users in the 2.19.0 release since the version will get updated in the URL and invalidate the cache:

https://github.com/mlflow/mlflow/blob/6bcec85dc190dde1257edd5f10e2d3db6dbf6d98/mlflow/server/js/src/shared/web-shared/model-trace-explorer/ModelTraceExplorerFrameRenderer.tsx#L77

checked with @TomeHirata and it seems like it works fine for him

**Review by TomeHirata — APPROVED** [review]
LGTM, verified that the exception marker is displayed correctly and warnings disappear.
image

**Pull request merged**